DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -2041,6 +2046,18 @@ globus_l_gfs_iRODS_recv(
         }
     }
 
+    result = globus_gridftp_server_get_recv_modification_time(op, &iRODS_handle->utime);
+    if(result != GLOBUS_SUCCESS)
+    {
+        // continue but don't modify utime
+        globus_gfs_log_message(GLOBUS_GFS_LOG_ERR,"iRODS: globus_gridftp_server_get_recv_modification_time error.\n");
```

Comment:
Is there any value in showing `result` in this error message?

Reply:
I changed it to print the return code.  I am not sure what might be returned as there are only #defines for GLOBUS_SUCCESS (0) and GLOBUS_FAILURE (-1).

Reply:
Consider asking the Globus team about whether other values can be returned.

If the only values ever returned are 0 or -1, then I don't see much value in logging it.

Thoughts?

Reply:
> Consider asking the Globus team about whether other values can be returned.
> 
> If the only values ever returned are 0 or -1, then I don't see much value in logging it.
> 
> Thoughts?

I went ahead and logged the error the way the Globus guy suggested. 

```
globus_gfs_log_result(GLOBUS_GFS_LOG_WARN, "iRODS: Error getting modtime, skipping: ", result); 
```

Reply:
They suggested warn and after seeing it I agreed that that is probably the better approach since this wouldn't be a fatal issue.

Here is the response I got back:

```
For a globus_result_t, anything other than GLOBUS_SUCCESS is error.

You can log the error:
{
     globus_gfs_log_result(GLOBUS_GFS_LOG_WARN, "Error getting modtime,
skipping:", result);
}

Or get the error string for your own logging:
{
     char *                              msg;
     msg = globus_error_print_friendly(globus_error_peek(result));
     ...
     globus_free(msg);
}

With some globus_result_t types, you can check the module-specific error
codes with additional globus_error_* api, but in this case there aren't
going to be any errors you can do anything about, so its safe to just
log the string.
```

I'm assuming from that response the the former will log the same error string as the second version.

Reply:
Sounds good to me.